koa-basic-auth/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for koa-basic-auth 2.x
+// Project: https://github.com/koajs/basic-auth
+// Definitions by: Tobias Wolff <https://github.com/Tobias4872/>
+// Definitions: https://github.com/Tobias4872/DefinitelyTyped
+
+import * as Koa from "koa";
+
+declare function auth(opts?: {
```

Comment:
I find it suspicious that `opts` is optional but the indivitual options aren't; could this be `name?: string` and `pass?: string`?